python-avd/pyavd/_utils/strip_empties.py

```diff
@@ -59,7 +63,39 @@ def strip_empties_from_list(data: list, strip_values_tuple: tuple = (None, "", [
     return new_data
 
 
-def strip_empties_from_dict(data: dict, strip_values_tuple: tuple = (None, "", [], {})) -> dict:
+def strip_empties_from_tuple(data: tuple, strip_values_tuple: tuple = (None, "", [], {}, ())) -> tuple:
+    """
+    strip_empties_from_tuple Remove entries with null value from a tuple.
+
+    Parameters
+    ----------
+    data : Tuple
+        data to filter
+    strip_values_tuple : tuple, optional
+        Value to remove from data, by default (None, "", [], {}, (),)
+
+    Returns:
+    -------
+    Any
+        Cleaned tuple with no strip_values_tuple
```

Comment:
We use `google` docstring convention:
```suggestion
    Recursively removes empty values from a tuple.
    
    Args:
          data: The tuple to filter.
          strip_values_tuple: A tuple of values to be considered empty and removed.
          
    Returns:
          A new tuple with all specified empty values recursively removed.
```

Reply:
Proposed changes to strip_empties 've been withdrawn